include/api/debug.h

```diff
@@ -77,7 +77,7 @@ static inline void debug_printUserState(void)
 
 static inline void debug_printTCB(tcb_t *tcb)
 {
-    printf("%40s\t", TCB_PTR_DEBUG_PTR(tcb)->tcbName);
+    printf("%lx %40s\t", (unsigned long int)tcb, TCB_PTR_DEBUG_PTR(tcb)->tcbName);
```

Comment:
Is this still useful?

Reply:
I think this question mostly to @hlyytine .

Reply:
Yes, with this we can dump pointer to TCB and it's name -- later on we can decode the thread names from the pointers, that is, we can have human readable names in kernelshark etc output.

Reply:
Hmm I just realized it's quite a feat to collect something that seL4 prints to UART to use with kernelshark... So in this context this is not needed and could be removed. But something to keep in mind that the human readable name is found from that struct.

Reply:
I think that actually it is better to keep it for now. And probably remove it when we adopt `human readable names` in kernelshark.